api/tests/app/conftest.py

```diff
@@ -764,33 +764,6 @@ def sample_invited_org_user(
     return create_invited_org_user(sample_organisation, sample_user)
 
 
-@pytest.fixture(scope='function')
```

Comment:
why gone

Reply:
refactored out in https://github.com/alphagov/notifications-api/commit/87cb6f25974d394a8fa1b8f9cd7abe8aa28e05a7#diff-aeffab693d3099ca885d19beb13c0ef5L773 —refactoring looks like a different way of creating their text fixtures. The old "def sample_x` funcs are on the way out.